### ISSUE TITLE
iocage fetch - is the output necessary?

### ISSUE THREAD

**Issue by xerovis** [iocage/iocage#236]
When I run iocage fetch, I get messages about how iocage is creating ZFS volumes. Is it really necessary to tell me what ZFS volumes iocage is creating? Do I really need to know that a whole bunch of folders and files are being downloaded/created? Is it really necessary to tell me that a whole bunch of stuff is going on under the hood? I just want to fetch a release and use iocage (for the most part). I can see the point of asking me what release to select but other than that, the rest is "extra" information that only serves to fill the page out, it doesn't really add anything. If things go wrong then I expect to see an error message but other than that, does the user really need to see all that other stuff?

**Comment by pr1ntf** [comment]
That "extra information under the hood" helps greatly when issues or bugs arise. That way if something breaks, we can track down where something went awry, rather than guess what part of the code actually broke. For instance, the `iocage fetch` function has 92 lines of code give or take some white space. There are several parts to that function, including downloading the release packages, creating the dataset, and creating basejails. Also, sometimes on slower systems or slower pools, dataset creation takes some time. There can be a period of time where nothing is happening on the console and could be mistaken for iocage hanging. Letting the user know what is going on also let's the user know iocage is doing it's thing. 


>does the user really need to see all that other stuff?
In short, yes. I understand that there are programs out there where you can change verbosity of output, but there really is not a large need for a quiet terminal output that we can see right now.
